openapi/datasource.yaml

```diff
@@ -223,6 +224,7 @@ components:
             - AzureBlobStorageConfig
             - BigQueryConfig
             - ClickHouseConfig
+            - Databricks
```

Comment:
should be `DatabricksConfig`

Reply:
Thanks for the catch! My bad, changing it now.